README-permission.md

```diff
@@ -0,0 +1,132 @@
+Permission module
+============
+
+Description
+-----------
+
+The permission module allows to ensure presence and absence of permissions and permission members.
+
+Features
+--------
+
+* Permission management
+
+
+Supported FreeIPA Versions
+--------------------------
+
+FreeIPA versions 4.4.0 and up are supported by the ipapermission module.
+
+
+Requirements
+------------
+
+**Controller**
+* Ansible version: 2.8+
+
+**Node**
+* Supported FreeIPA version (see above)
+
+
+Usage
+=====
+
+Example inventory file
+
+```ini
+[ipaserver]
+ipaserver.test.local
+```
+
+
+Example playbook to make sure permission "NAME" is present:
+
+```yaml
+---
+- name: Playbook to manage IPA permission.
+  hosts: ipaserver
+  become: yes
+
+  tasks:
+  - ipapermission:
+      ipaadmin_password: SomeADMINpassword
+      name: NAME
+      # Add needed parameters here
+```
+
+Example playbook to make sure permission "NAME" member PARAMETER2 VALUE is present:
+
+```yaml
+---
+- name: Playbook to manage IPA permission PARAMETER2 member.
+  hosts: ipaserver
+  become: yes
+
+  tasks:
+  - ipapermission:
+      ipaadmin_password: SomeADMINpassword
+      name: NAME
+      PARAMETER2: VALUE
+      action: member
+```
+
+
+Example playbook to make sure permission "NAME" member PARAMETER2 VALUE is absent:
+
+
+```yaml
+---
+- name: Playbook to manage IPA permission PARAMETER2 member.
+  hosts: ipaserver
+  become: yes
+
+  tasks:
+  - ipapermission:
+      ipaadmin_password: SomeADMINpassword
+      name: NAME
+      PARAMETER2: VALUE
+      action: member
+      state: absent
+```
+
+
+Example playbook to make sure permission "NAME" is absent:
+
+```yaml
+---
+- name: Playbook to manage IPA permission.
+  hosts: ipaserver
+  become: yes
+
+  tasks:
+  - ipapermission:
+      ipaadmin_password: SomeADMINpassword
+      name: NAME
+      state: absent
+```
+
+
+MORE EXAMPLE PLAYBOOKS HERE
+
+
+Variables
+---------
+
+ipapermission
+-------
+
+Variable | Description | Required
+-------- | ----------- | --------
+`ipaadmin_principal` | The admin principal is a string and defaults to `admin` | no
+`ipaadmin_password` | The admin password is a string and is required if there is no admin ticket available on the node | no
+`name` \| `ALIAS` | The list of permission name strings. | yes
+`PARAMETER1` \| `API_PARAMETER_NAME` | DESCRIPTION | BOOL
+`PARAMETER2` \| `API_PARAMETER_NAME` | DESCRIPTION | BOOL
```

Comment:
The variables accepted by the module should be added here. You can use the same values used in the source code documentation.

Reply:
I updated the docs and I think I have the minimum now.